### PR TITLE
Adding mlgb

### PR DESCRIPTION
**Pull request by UlionTse**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by UlionTse** [comment]
@conda-forge/help-python Error is from windows environment itself ?
mlgb_conda_error_1

**Comment by UlionTse** [comment]
@conda-forge/help-python This is ready to review, thanks.

**Comment by UlionTse** [comment]
@conda-forge/help-python ready for review

**Comment by UlionTse** [comment]
@conda-forge/help-python ready for review !!!

File: recipes/mlgb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mlgb-{{ version }}.tar.gz
+  sha256: 68b22a5663bcb908b0f02294508c77ae60c527b295b1a0b3b2aeb510a4c23fef
+
+build:
+  skip: True  # [win, py<38]
```
Comment (ocefpaf) [inline recipes/mlgb/meta.yaml:13]: This one should be noarch. It is OK to fail on Windows due to missing dependencies in that case.

Comment (UlionTse) [reply recipes/mlgb/meta.yaml:13]: @ocefpaf Okay

**Comment by UlionTse** [comment]
@ocefpaf Thanks.

**Pull request merged**